scoap3/articles/tasks.py

```diff
@@ -122,6 +124,20 @@ def check_doi_registration_time(obj):
     return False, "DOI not found in our system."
 
 
+def check_authors_affiliation(article_id):
```

Comment:
why pass `article_id` if you need to fetch whole `article` object, already there